test/integration/react-streaming-and-server-components/app/pages/index.server.js

```diff
@@ -10,7 +10,7 @@ export default function Index({ header }) {
     <div>
       <Head>
         <meta name="rsc-title" content="index" />
-        <title>hello, {envVar}</title>
+        <title>{`hello, ${envVar}`}</title>
```

Comment:
This was fixed so we should have the test to make sure it doesn't regress.

Reply:
This is for fixing a react 18.2 warning, which is missed in react 18.2 version bump PR